packages/cnpj-gen/tests/Specs/CnpjGenerator.isolated.spec.php

```
@@ -0,0 +1,102 @@
+
+            return parent::generate($options, $format, $prefix, $type);
+        }
+    }
+
+    function mockSequenceGenerator(string ...$returns): void
+    {
+        $mockingException = Mockery::mock('alias:' . SequenceGenerator::class)
+            ->shouldReceive('generate');
+
+        if (!$mockingException instanceof CompositeExpectation) {
+            throw new RuntimeException('Unexpected Mockery expectation type for SequenceGenerator::generate.');
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '66', column '23'). (undefined)

(MissingImport)